src/Kmd.Logic.Gateway.Automation/ValidateProduct.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Kmd.Logic.Gateway.Automation.Gateway;
+using YamlDotNet.Serialization;
+
+namespace Kmd.Logic.Gateway.Automation
+{
+    public class ValidateProduct
+    {
+        private readonly MediaConfiguration _mediaConfiguration = new MediaConfiguration()
+        {
+            MaxLogoSizeBytes = 1000000,
+            MaxMarkDownDocumentSizeBytes = 1000000,
+            MaxOpenApiSpecSizeBytes = 20971520,
+            AllowedLogoMimeTypes = new[] { ".png", ".jpeg" },
+            AllowedMarkdownDocumentExtension = ".md",
+        };
+
+        private IList<PublishResult> publishResults;
+
+        public IList<PublishResult> IsProductValid(string folderPath)
+        {
+            this.publishResults = new List<PublishResult>();
+
+            using (var publishYml = File.OpenText(Path.Combine(folderPath, @"publish.yml")))
```

Comment:
Use simple form of using https://docs.microsoft.com/pl-pl/dotnet/csharp/whats-new/csharp-8#using-declarations

Reply:
Done